templates/industrial/index.html

```diff
@@ -0,0 +1,695 @@
+          alt="",
+          height="250",
+          width="248",
+          hi_def=True,
+        ) | safe
+      }}
+    </div>
+  </div>
+  <div class="u-fixed-width">
+    <hr class="p-separator" />
+  </div>
+
+  <!-- Exciting HMIs and POS -->
+  <div class="row u-equal-height">
+    <div class="col-6 u-vertically-center u-align--center u-hide--small">
+      {{
+        image(
+          url="https://assets.ubuntu.com/v1/32e0b197-digital-kiosk.svg",
+          alt="",
+          height="250",
+          width="178",
+          hi_def=True,
+        ) | safe
+      }}
+    </div>
+    <div class="col-6">
+      <h2>Exciting HMIs and POS</h2>
+      <p class="p-heading--four">Fast, fluid and designer-friendly</p>
+      <ul>
+        <li>Implement robust digital signage and kiosk solutions</li>
+        <li>Upgrade your terminals securely, with reliable OTA updates</li>
+        <li>Benefit from a rich hardware and software ecosystem</li>
+        <li>Millions of smart displays already deployed on Ubuntu</li>
+        <li>Create rich graphical interfaces with Qt and Flutter</li>
+      </ul>
+      <ul class="p-inline-images u-no-margin--bottom u-align--left">
+      
+        <li class="p-inline-images__item u-no-margin--left">
+          <a href="https://snapcraft.io/flutter">
+            {{
+              image(
+                url="https://assets.ubuntu.com/v1/7475d014-logo_flutter_1080px_clr.svg",
+                alt="Flutter",
+                height="88",
+                width="88",
+                hi_def=True,
+                loading="lazy",
+                attrs={"class": "p-inline-images__logo"},
+              ) | safe
+            }}
+          </a>
+        </li>
+        <li class="p-inline-images__item u-no-margin--left">
+          {{
+            image(
+              url="https://assets.ubuntu.com/v1/4076f787-QT-software.png",
+              alt="QT",
+              height="88",
+              width="120",
+              hi_def=True,
+              loading="lazy",
+              attrs={"class": "p-inline-images__logo"},
+            ) | safe
+          }}
+        </li>
+        <li class="p-inline-images__item u-no-margin--left">
+          {{
+            image(
+              url="https://assets.ubuntu.com/v1/24a4d66d-HTML5_logo_and_wordmark.svg",
+              alt="HTML5",
+              height="88",
+              width="66",
+              hi_def=True,
+              loading="lazy",
+              attrs={"class": "p-inline-images__logo"},
+            ) | safe
+          }}
+        </li>
+      </ul>          
+      <a href="/internet-of-things/digital-signage">
+        Learn more&nbsp;&rsaquo;
+      </a>
+    </div>
+  </div>
+
+  <div class="u-fixed-width">
+    <hr class="p-separator" />
+  </div>
+
+ <div class="row u-equal-height">
+    <div class="col-6">
+      <h2>Do you have other use cases in mind?</h2> 
+      <!-- TODO which contact us-->
```

Comment:
Requested a form from Lucia in the copy doc. But, I think that the iot contact us is fine but we will have to add a conditional to page to change the wording from automotive to industry. Then change this link to `?product=industry`